app/lib/bk/compat/parsers/bitrise/slack.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module BK
+  module Compat
+    module BitriseSteps
+      # Handles the translation of slack notifications
+      class Translator
+        def translate_slack(inputs)
+          warnings = generate_warnings(inputs)
+
+          if error_config?(inputs)
+            create_dual_notifications(inputs, warnings)
+          else
+            create_single_notification(inputs, warnings)
+          end
+        end
+
+        private
+
+        def error_config?(inputs)
+          inputs.keys.any? { |k| k.end_with?('_on_error') && !inputs[k].to_s.empty? }
```

Comment:
this condition means that it will create two notification settings even when the `_on_error` settings are for unsupported options